mod.cpp

```diff
@@ -1,12 +1,12 @@
 name = "ACE3";
 picture = "logo_ace3_ca.paa";
-actionName = "Github";
-action = "https://github.com/KoffeinFlummi/ACE3";
-description = "ACE3 - Version 3.0.0";
+actionName = "GitHub";
+action = "https://github.com/acemod/ACE3";
+description = "ACE3 – Version 3.0.0";
```

Comment:
@esteldunedain, done. Haven't tested it either way yet, so I am following your suggestion.